content/_data/changelogs/lts.yml

```diff
@@ -2912,16 +2902,12 @@
       pull: 4242
       authors:
         - daniel-beck
+      references:
+        - pull: 4040
       message: |-
-        Stop bundling Maven Plugin and Subversion Plugin with Jenkins.
+        Stop bundling Maven Plugin, Subversion Plugin, translation, ssh-credentials, credentials, and ssh-slaves plugins inside Jenkins war file.
```

Comment:
Most of these are not the user-friendly names, but artifactId.

Most of these are irrelevant to note here as a major change, since there's no situation since 2.0 that they would be unbundled. (Maven and Subversion are only notable because of the unlikely but possible compatibility impact.)